vnext/src/RNTester/js/components/RNTesterExampleList.windows.js

```diff
@@ -1,8 +1,6 @@
 /**
- * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Same here. We must not remove Facebook copyright notice. Why not just remove the 'All rights reserved' from Microsoft copyright notice?